.github/workflows/docs.yml

```
@@ -31,6 +38,12 @@ jobs:
           set: |
             *.cache-from=type=gha,scope=docs
             *.cache-to=type=gha,scope=docs,mode=max
+            docs-export.args.DOCS_BASEURL=${{ steps.pages.outputs.base_path }}
```

Related to https://github.com/distribution/distribution/pull/4102/files#r1359481280

Then you just need to send the `DOCS_BASEURL` env:

```suggestion
          env:
            DOCS_BASEURL: ${{ steps.pages.outputs.base_path }}
```